cypress/integration/page.spec.js

```diff
@@ -1,7 +1,132 @@
-// WRITE TESTS HERE
+/// <reference types="cypress" />
 
 describe('Page', () => {
-  it('should be visitable', () => {
-    cy.visit('/');
+  const apiUrl = 'https://mate.academy/students-api/'
+
+  beforeEach(() => {
+    cy.visit('/')
+  });
+
+  it('user details should be updated after selecting a different user', () => {
+    cy.intercept(`${apiUrl}` + 'users/5', { fixture: 'user5' });
+    cy.intercept(`${apiUrl}` + 'users/6', { fixture: 'user6' });
+
+    cy.get('button')
+      .contains('5')
```

Comment:
Here you still work with todos from the real server

Reply:
fixed by adding mocking to the before each hook